rego/Gmail.rego

```diff
@@ -1904,12 +1904,12 @@ tests contains {
         "While we are unable to determine the state from the logs, ",
         "the default setting is compliant; manual check recommended."
     ]),
-    "ActualValue": {"EMAIL_SPAM_ALLOWLIST": "No relevant event in the current logs"},
+    "ActualValue": {"GmailIpWhitelistSpamFilteringProto is_empty": "No relevant event in the current logs"},
     "RequirementMet": true,
     "NoSuchEvent": true
 }
 if {
-    Events := utils.FilterEventsNoOU(LogEvents, "EMAIL_SPAM_ALLOWLIST")
+    Events := utils.FilterEventsNoOU(LogEvents, "GmailIpWhitelistSpamFilteringProto is_empty")
```

Comment:
These events _do_ have an OU associated with them. So we'd actually want `FilterEventsOU` here. NOTE: currently, you cannot apply this setting to a child OU so it technically works as is. This would be more of a future proofing thing.
![image](https://github.com/user-attachments/assets/7979e5ae-c4a4-43e6-820a-c866c9f4ebde)

Reply:
@adhilto One of the things I changed was that the old rego pointed to the Domain Name instead of the Org_Unit_Name, which is why the previous version didn't use an OU. I can make the change to future-proof though!